cockatrice/src/userconnection_information.cpp

```diff
@@ -0,0 +1,72 @@
+#include "userconnection_information.h"
+
+UserConnection_Information::UserConnection_Information()
+{
+}
+
+void UserConnection_Information::addNewServer(QString _saveName, QString _serv, QString _port, QString _username, QString _password, bool _savePassword)
+{
+    QSettings settings("Cockatrice", "server_details");
```

Comment:
You may want to prefer serversettings instead of using QSettings directly.
Server settings are currently managed here: https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/settings/serverssettings.h , just adapt it to your needs